.github/workflows/docker-tagged.yml

```diff
@@ -0,0 +1,47 @@
+name: Docker Tagged Build
+
+on:
+  push:
+    tags:
+      - "*"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      DOCKER_NAMESPACE: "openfaas"
+      DOCKER_IMAGE_NAME: "nats-connector"
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+
+      - name: Get TAG
+        id: get_tag
+        run: echo ::set-output name=TAG::${GITHUB_REF#refs/tags/}
+
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v1
+        with:
+          version: latest
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v1 
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+          registry: docker.io
```

Comment:
Please change to `ghcr.io` - we are moving all images as part of the larger program.

https://trello.com/c/uqaun5lI/172-4th-nov-development-call

We are also submitting 3 tags, can you amend?

Reply:
Cool yes - sorry I saw that call too late to join, will update according to those notes